repose-aggregator/tests/functional-tests/src/integrationTest/groovy/features/core/powerfilter/DestinationRouterJMXTest.groovy

```diff
@@ -62,32 +59,4 @@ class DestinationRouterJMXTest extends ReposeValveTest {
         then:
         repose.jmx.getMBeanAttribute(DESTINATION_ROUTER_TARGET, "Count") == (target + 1)
     }
-
-
-    def "when requests match destination router target URI, should increment DestinationRouter mbeans for all endpoints"() {
```

Comment:
That is right. We no longer have the `ALL` metrics that were being managed by the custom metric classes that were removed. I was actually surprised to see that the Dropwizard metrics library doesn't seem to support any metric aggregation across metrics.

So, since that aggregation is just another metric, we could still manually report it, or leave it to the user to aggregate the metrics themselves. Right now, that's gone, so I removed the tests.

If we still want the `ALL` metrics, I would suggest not forcing them onto the user, but providing them as an option. So write aggregation classes for each type of metric and make them available, but don't make them the only way to report metrics (because that limits our support for the metrics library).